docker/puppetserver/Dockerfile

```diff
@@ -100,16 +101,16 @@ ENV RUBYOPT=-r/puppetserver/docker/ruby-docker-copy-patch
 
 RUN lein clean && \
     lein install && \
-    EZBAKE_ALLOW_UNREPRODUCIBLE_BUILDS=true EZBAKE_NODEPLOY=true COW=base-bionic-amd64.cow MOCK='' GEM_SOURCE=https://rubygems.org lein with-profile ezbake ezbake local-build && \
+    EZBAKE_ALLOW_UNREPRODUCIBLE_BUILDS=true EZBAKE_NODEPLOY=true COW=base-bionic-${TARGETARCH}.cow MOCK='' GEM_SOURCE=https://rubygems.org lein with-profile ezbake ezbake local-build && \
```

Comment:
Have you verified that this works? I don't think EZBake supports ARM

https://github.com/puppetlabs/packaging/blob/374e704ae9683712fdbca71f50699b33c8d494fa/lib/packaging/platforms.rb#L196

Reply:
I think the only arm support we have is for M1 macs - https://github.com/puppetlabs/packaging/blob/374e704ae9683712fdbca71f50699b33c8d494fa/lib/packaging/platforms.rb#L112

Reply:
> Have you verified that this works? I don't think EZBake supports ARM
> 
> https://github.com/puppetlabs/packaging/blob/374e704ae9683712fdbca71f50699b33c8d494fa/lib/packaging/platforms.rb#L196

@Iristyle On that line you linked, I can see that `aarch64` is listed (which is just another name for `arm64`). Is there something else that would prevent EZBake from working?